test/test_plugin_systemd.rb

```diff
@@ -11,8 +11,6 @@ class TestPluginSystemd < TestIntegration
 
   def setup
     skip_unless :linux
-    skip_unless :unix
-    skip_unless_signal_exist? :TERM
```

Comment:
Unix is redundant if checking Linux (`UNIXSocket` will be defined), TERM exists on Linux.